### PR TITLE
Update Org Images To Render Dynamically

### LINKED ISSUE #2475: Update the Organization images list under the Citizen Engagement page so it's dynamic
### Dependency
- #2459

### Overview
We need to update the Citizen Engagement Page so that the data is dynamically rendered using liquid.

### Action Items
- [x] Under the section "Organizations We Work With," update the html so that the images are rendered from the data files 
    - [x] Use liquid to loop through the list of images under the [partners.yml](https://github.com/hackforla/website/blob/gh-pages/_data/internal/partners.yml) file 
    - [x] Check the program-area property. If the value is "Citizen Engagement" the image should be rendered
    - [x] Be sure to include the `alt` attribute in the loop
      - [x] The alt text for each image will simply be the `name` of the organization
- [x] Ensure that the page still looks and functions the same after the refactor

#### Merge Team, Technical Lead, or Product Manager
- [ ] Once the pull request associated with this issue is merged, go to issue #3328 and check if all the issues listed under the `Dependency` section are closed. If they are all closed, move that issue (#3328) into the `New Issue Approval` column, remove the `Dependency` label on it, and add a `ready for dev lead` label to it.

### Resources/Instructions
[partners.yml](https://github.com/hackforla/website/blob/gh-pages/_data/internal/partners.yml)
[Citizen Engagement Page](https://www.hackforla.org/citizen-engagement)
[citizen-engagement.html](https://github.com/hackforla/website/blob/gh-pages/pages/citizen-engagement.html)
[Using liquid with jekyll](https://jekyllrb.com/docs/liquid/)

### PR DESCRIPTION
**Pull request by Skydodle**
Fixes #2475 

### What changes did you make and why did you make them ?

  - On citizen-engagement page under "Organizations We Work With" section, replaced the hard-coded image elements with a liquid loop so that the images render dynamically based on the data file partner.yml, specifically for organizations that contain "Citizen Engagement" in their program-area property
  - Images count decreased after change, this is due to the partner.yml data file will be updated later after this issue is completed. Confirmed with Justin and Bonnie.

### Screenshots of Proposed Changes Of The Website  (if any, please do not screen shot code changes)

<details>
<summary>Visuals before changes are applied</summary>

Screenshot 2023-03-01 at 12 46 01 AM

</details>

<details>
<summary>Visuals after changes are applied</summary>
  
Screenshot 2023-03-01 at 1 22 24 AM

</details>

### CODE REVIEW

**Comment by jdingeman** [comment]
Hi @Skydodle - be sure to add "Fixes" before the issue number so the github bot can pull the labels from it.

**Comment by Skydodle** [comment]
@jdingeman Gotcha thank you so much for the reminder!

**Comment by jdingeman** [comment]
ETA: EOD Sunday, March 12
Availability: today, 3pm - 6pm

**Comment by chrismenke45** [comment]
Review ETA: 3/13 12pm
Availability: 3/13 8am-12pm

**Comment by steven-positive-tran** [comment]
Review ETA: 3/14 11pm
Availability: 3/13, 3/14

**Review by steven-positive-tran — APPROVED** [review]
Branches look good,

The citizen Engagement page is updating when I edit the partners.yml page ( I have to restart the docker server everytime I edit the yml page. Is there a reason for that? )

**Review by chrismenke45 — APPROVED** [review]
Looks good! The codes concise and works as expected.

**Review by jdingeman — APPROVED** [review]
Hey @Skydodle, this looks good. Nice job!

**Comment by Skydodle** [comment]
> 

@steven-positive-tran I'm not 100% sure but my guess is that partner.yml is a data file and Docker creates a container based on the data file prior to change. Similar to my experience using Docker with a SQL database, whenever there's a change in the database I would also need to restart Docker to create a new container image unless there's a migration support system set up in the application-level database library. 

In short, unlike the frontend changes, Docker does not automatically reflect database level changes is my guess.
